### PR TITLE
Note about SNTP for Shelly

### PR DESCRIPTION
**Pull request by chemelli74**
## Proposed change

Add info about the need of a working SNTP for some features, like power consumption.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/shelly.markdown [hunk 1]
```diff
@@ -191,6 +191,11 @@ The firmware limits the transition time to 5 seconds.
 
 </div>
 
+## Additional info
+
+Shelly's devices rely on [SNTP](https://en.wikipedia.org/wiki/Network_Time_Protocol) for some of their features, like power measurement.
```
Comment (ludeeus) [inline source/_integrations/shelly.markdown]:
```suggestion
Shelly devices rely on [SNTP](https://en.wikipedia.org/wiki/Network_Time_Protocol#SNTP) for features like power measurement.
```

File: source/_integrations/shelly.markdown [hunk 2]
```diff
@@ -191,6 +191,11 @@ The firmware limits the transition time to 5 seconds.
...
+Please ensure that the configured server is reacheable.
```
Comment (ludeeus) [inline source/_integrations/shelly.markdown]: How? Where?

Comment (chemelli74) [reply source/_integrations/shelly.markdown]: Detailed

File: source/_integrations/shelly.markdown [hunk 3]
```diff
@@ -191,6 +191,11 @@ The firmware limits the transition time to 5 seconds.
 
 </div>
 
+## Additional info
+
+Shelly devices rely on [SNTP](https://en.wikipedia.org/wiki/Network_Time_Protocol#SNTP) for features like power measurement.
+Please check from the device Web UI that the configured server is reacheable.
```
Comment (frenck) [inline source/_integrations/shelly.markdown:197]: - `reacheable` -> `reachable`

Comment (chemelli74) [reply source/_integrations/shelly.markdown:197]: Opened PR to fix the typo ;-)

Comment (frenck) [reply source/_integrations/shelly.markdown:197]: Thanks, @chemelli74 👍

**Pull request merged**